docs/interactions.rst

```diff
@@ -0,0 +1,274 @@
+Interactions
+============
+
+Pincer makes it extremely easy to create an interaction. All you need to do is create a method with the `@command` decorator.
+
+.. note::
+    Although purely functional bots are possible, its recommended to inherit from the `Client` class as seen in the example below.
+
+.. code-block:: python
+
+  from pincer import Client
+  from pincer.commands import command
+
+  class Bot(Client):
+    @command
+    async def some_command(self):
+      ...
+
+  bot = Bot("TOKEN")
+  bot.run()
+
+This registers a command called "some_command". Its pretty useless right now so lets take a closer look at what else you can do.
+
+.. note::
+    If you aren't seeing your command register, its likely because it takes up to one hour for a global slash command.
+    Specify the guild in the command decorator to have application commands register instantly.
+
+To register your command a guild do:
+
+.. code-block:: python
+
+    @command(guild=SOME_NUMBER)
+    async def some_command(self):
+        ...
+
+Sending messages to the user is extremely easy. What you want to return is inferred by the object's return type. A :class:`str` can be returned to send a text message.
+Heres a simple ping command.
+
+.. code-block:: python
+
+    @command
+    async def ping(self):
+        return "pong"
+
+
+If you need access to more information, you can pass in the :data:`ctx` object.
```

Comment:
Suggestion, what if we put a link to the class here?

```suggestion
If you need access to more information, you can pass in the :class:`ctx <~pincer.objects.message.context.MessageContext>` object.
```

Reply:
Good idea